source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -723,7 +718,12 @@ The event API here is assumed to be like the standard `Python Event
         try:
             # The connection is null if this is the first check. It's closed if there was an error during the previous
             # check or the previous check was cancelled.
-            # NOTE: helloCommand is defined in __init__() above
+
+            if previousDescription.helloOk:
```

Comment:
To skip storing on the ServerDescription:
```suggestion
            if helloOk:
```

Reply:
Fixed.